src/js/archive.js

```diff
@@ -29,7 +29,13 @@ function populateArchive() {
 
       // if there are releases prior to the 'latest' one (i.e. archived releases)...
       if (typeof releasesJson[0] !== 'undefined') {
-        buildArchiveHTML(releasesJson);
+        loadJSON(repoName, 'jck', function(response_jck) {
+          var jckJSON = {}
+          if (response_jck !== null){
+            jckJSON = JSON.parse(response_jck)
+          }
+          buildArchiveHTML(releasesJson, jckJSON);
```

Comment:
Couldn't we make JCK status part of the releasesJson?  That is, merge it?

Reply:
Not very easily. It would make our website-backend job a lot more complicated